__tests__/app.test.js

```diff
@@ -92,6 +92,69 @@ describe("GET /api/reviews", () => {
         });
       });
   });
+  it("should return status 200, responds with review selected from category query value", () => {
+    return request(app)
+      .get("/api/reviews?category=social deduction")
+      .expect(200)
+      .then((response) => {
+        const reviewsArray = response.body.reviews;
+        expect(reviewsArray.length).toBe(11);
+      });
+  });
+  it("should return status 200, responds with review ordered by query value", () => {
+    return request(app)
+      .get("/api/reviews?order=ASC")
+      .expect(200)
+      .then((response) => {
+        const reviewsArray = response.body.reviews;
+        expect(reviewsArray).toBeSortedBy("created_at", { descending: false });
+      });
+  });
+  it("should return status 200, responds with review sorted by query value", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=comment_count")
+      .expect(200)
+      .then((response) => {
+        const reviewsArray = response.body.reviews;
+        expect(reviewsArray).toBeSortedBy("comment_count", {
+          descending: true,
+        });
+      });
+  });
+  it("should return status 200, should handle all query values simultaneously", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=votes&order=ASC&category=social deduction")
+      .expect(200)
+      .then((response) => {
+        const reviewsArray = response.body.reviews;
+        expect(reviewsArray.length).toBe(11);
+        expect(reviewsArray).toBeSortedBy("votes", { descending: false });
+      });
+  });
+  it("should return status 404, when passed category query value that does not exist", () => {
+    return request(app)
+      .get("/api/reviews?category=CHEESE")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.message).toBe("invalid category query");
+      });
+  });
+  it("should return status 400, when passed invalid sort_by query", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=BISCUITS")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.message).toBe("invalid sort query");
+      });
+  });
+  it("should return status 400, when passed invalid order query", () => {
+    return request(app)
+      .get("/api/reviews?order=BISCUITS")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.message).toBe("invalid sort order");
+      });
+  });
```

Comment:
Really nice! Can we consider one my scenario:
What will happen if the category query is valid and the category exists but there happens to be no reviews currently associated with that category? This isn't an error as nothing has gone wrong, there's just no responses.